_articles/system-firmware.md

```diff
@@ -55,6 +56,14 @@ sudo system76-firmware-cli schedule
 
 Reboot the system using `sudo systemctl reboot` to proceed with the update, or use `sudo system76-firmware-cli unschedule` to cancel the update.
 
+### Transitioning to Open Firmware
```

Comment:
Steps should be added for transitioning back

Reply:
We'll also need to include expected feature differences between the two so they know what to expect before transitioning.

Reply:
Added the command to transition back and some expected differences in https://github.com/system76/docs/pull/460/commits/eae2c0032bab391290a45c9394caac5774171b29, let me know if either of you think anything important is missing or if anything in the list of differences looks inaccurate.